doc/developers/contributing.rst

```diff
@@ -43,6 +43,33 @@ ticket to the
 also welcome to post feature requests or pull requests.
 
 
+Ways to contribute
+==================
+
+There are many ways to contribute to scikit-learn, with
+the most common ones being contribution of code or documentation to
+the project. Improving the documentation is no less important than
+improving the library itself.
+If you find a typo in the documentation, or have made improvements, do not
+hesitate to send an email to the mailing list or submit a GitHub pull request.
```

Comment:
Perhaps we should say "preferably" before "submit" so that people know this is better than sending an email where possible.